akka-kinesis-kcl/src/main/scala/com/github/j5ik2o/ak/kcl/stage/KCLSourceStage.scala

```diff
@@ -197,10 +197,15 @@ class KCLSourceStage(
 
       private var worker: Worker = _
 
+      private var shardId: String = _
+
       private var buffer: Queue[RecordSet] = Queue.empty[RecordSet]
 
       private val onInitializeCallback: AsyncCallback[InitializationInput] = getAsyncCallback { initializationInput =>
-        log.debug(s"onInitializeCallback: initializationInput = $initializationInput")
+        log.debug(
+          s"onInitializeCallback: initializationInput = shardId:${initializationInput.getShardId}, extendedSequenceNumber:${initializationInput.getExtendedSequenceNumber}, pendingCheckpointSequenceNumber:${initializationInput.getPendingCheckpointSequenceNumber}"
+        )
+        shardId = initializationInput.getShardId
```

Comment:
Print input details, because the InitializationInput does not implement toString().